### PR TITLE
Mdct 2746 - add sidebar component

### PR DESCRIPTION
**Pull request by britt-mo**
### Description
Added the sidebar component at pretty much the most basic level. No routing, no ReportProvider. I've set isHidden to false so you should see the sidebar appear when you go to /standard

### Related ticket(s)

MDCT-2764

---
### How to test
spin up locally, go to /standard, see the sidebar

Screenshot 2023-08-10 at 9 42 12 AM

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by karla-vm — APPROVED** [review]
💯 looks awesome, Britt!

File: services/ui-src/src/components/reports/ReportPageWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { Flex } from "@chakra-ui/react";
+import { PageTemplate, Sidebar, StandardReportPage } from "components";
+// utils
+
+import React from "react";
+import { mockStandardReportPageJson } from "utils/testing/mockForm";
+
+export const ReportPageWrapper = () => {
+  return (
+    <PageTemplate type="standard">
```
Comment (ntsummers1) [inline services/ui-src/src/components/reports/ReportPageWrapper.tsx:11]: I think we can swap this type to report, it'll make the display of the page much nicer. To do that you'll need to change `type?: "standard"` on line 25 of PageTemplate.tsx to `type?: "standard" | "report";`

File: services/ui-src/src/components/reports/ReportPageWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// components
+import { Flex } from "@chakra-ui/react";
+import { PageTemplate, Sidebar, StandardReportPage } from "components";
+// utils
+
+import React from "react";
```
Comment (ntsummers1) [inline services/ui-src/src/components/reports/ReportPageWrapper.tsx:6]: Can move this to the top of the file

File: services/ui-src/src/components/app/AppRoutes.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ export const AppRoutes = () => {
       <ScrollToTopComponent />
       <Routes>
         <Route path="/" element={<HomePage />} />
+        <Route path="/standard" element={<ReportPageWrapper />} />
```
Comment (ntsummers1) [inline services/ui-src/src/components/app/AppRoutes.tsx:13]: I'm ok with doing this for now for easy debugging/creation of components. Is everyone else ok with it?

Comment (karla-vm) [reply services/ui-src/src/components/app/AppRoutes.tsx:13]: Yep! I spoke to Britt yesterday about it, we'll just handle that once we have real components out there.

**Review by ntsummers1 — APPROVED** [review]

**Pull request merged**